Assets/Scripts/UI/LevelCard.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LevelCard : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject levelCard;
+    [SerializeField]
+    private string cardName;
+
+    public GameObject getLevelCard()
```

Comment:
These methods should be properties instead (e.g. LevelCard => ... for this one)

Reply:
I learn something new every PR. Fixed